test/DotNetty.Tests.Common/TestBase.cs

```diff
@@ -0,0 +1,30 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace DotNetty.Tests.Common
+{
+    using System;
+    using System.Diagnostics.Tracing;
+    using DotNetty.Common.Internal.Logging;
+    using Microsoft.Practices.EnterpriseLibrary.SemanticLogging;
+    using Xunit.Abstractions;
+
+    public abstract class TestBase : IDisposable
+    {
+        protected readonly ITestOutputHelper Output;
```

Comment:
Just for my benefit here, what's the purpose of this new base class and the `ITestOutputHelper` ?